packages/users/src/server/server.ts

```diff
@@ -76,9 +73,12 @@ export class UsersServer {
 	async deleteUser(user: User): Promise<void> {
 		try {
 			const queryOptions:QueryOptions = {
-				sql: "DELETE FROM users where userId=?"
+				namedPlaceholders:true,
+				sql: "DELETE FROM users where userId=:userId"
 			}
-			const values:number[] = [user.userId];
+			const values = {
+				userId:user.userId
+			};
 			await this.conn.query(queryOptions,values);
```

Comment:
Am discutat sa incerci sa pui direct obiectul in loc sa pui values. Ai incercat?

```js
await this.conn.query(queryOptions, user);
```